std/datetime/stopwatch.d

```diff
@@ -82,6 +82,22 @@ struct StopWatch
 {
 public:
 
+    /// Measure a time in milliseconds, microseconds, or nanoseconds
+    @safe nothrow @nogc unittest
+    {
+        auto sw = StopWatch(AutoStart.no);
+        sw.start();
+        // ... Insert operations to be timed here ...
+        sw.stop();
+
+        long msecs = sw.peek().total!"msecs";
+        long usecs = sw.peek().total!"usecs";
+        long nsecs = sw.peek().total!"nsecs";
```

Comment:
Idiomatic D doesn't use parenthesis in UFCS calls -> `sw.peek.total!"msecs"`.
(Users will most likely use exactly what is written here, so it might make sense to be picky about the style of the first example)

Reply:
Makes sense